t/obs/systemsmanagement:Agama:Devel/files_iso.lst

```diff
@@ -1,5 +1 @@
-agama-installer.aarch64-9.0.0-openSUSE-Build8.7.iso
-agama-installer.ppc64le-9.0.0-openSUSE-Build8.7.iso
-agama-installer.s390x-9.0.0-openSUSE-Build8.7.iso
-agama-installer.x86_64-9.0.0-openSUSE-Build8.7.iso
-
+agama-installer.s390x-9.0.0-openSUSE-Build17.9.iso
```

Comment:
all the rest files in `t`, which are not in `base` nor `s390x` folder should be removed if you add batches.

Reply:
Yes, removed.